source/index.css

```diff
@@ -20,4 +20,40 @@ canvas {
     @include aspect-ratio(160, 90);
 }
 
+.player-score {
+    position: relative;
+    text-align: center;
+    text-decoration: underline;
+    font-weight: bold;
+}
+
+.player-score .foreground-score {
+    position: relative;
+    z-index: 1;
+}
+
+.player-score .shadow-score {
```

Comment:
Niiice! I was wondering how you got the shadow. You really engineer a nice system here!

It doesn't have to be complicated, though. May I introduce you to the `text-shadow` property in CSS? http://www.w3schools.com/cssref/tryit.asp?filename=trycss3_text-shadow